pkg_ci.pt.Rmd

```diff
@@ -111,8 +111,8 @@ Após a transferência para a organização "ropensci" no GitHub pertencente a r
 Você pode encontrar o status dessas compilações em `https://ropensci.r-universe.dev/ui#packages` e na seção [status do commit](https://ropensci.org/blog/2021/09/03/runiverse-docs/#how-it-works). A compilação do site usará o seu arquivo config do pkgdown, se você tiver um, exceto para o estilo que usará o pacote [`rotemplate`](https://github.com/ropensci-org/rotemplate/).
 Se sua documentação incluir código que dependa, por exemplo, de credenciais, veja aqui como garantir que os documentos pkgdown sejam renderizados da melhor maneira possível.
 
-- Para exemplos de funções, use a tag roxygen2 `examplesIf` com a variável `IN_PKGDOWN`, por exemplo, `#' @examplesIf identical(Sys.getenv(“IN_PKGDOWN”), “true”)`
-- Para vinhetas, pré-compile se forem necessárias ferramentas/dados/credenciais especiais que não estão disponíveis em servidores de compilação genéricos (consulte https://ropensci.org/blog/2019/12/08/precompute-vignettes/) ou use a variável `IN_PKGDOWN` com a opção knitr eval, por exemplo
+- Para exemplos de funções, use a tag roxygen2 `examplesIf` com a variável `IN_PKGDOWN`, por exemplo, `#' @examplesIf identical(Sys.getenv(“IN_PKGDOWN”), “true”)`. Exemplo: [gtexr](https://github.com/ropensci/gtexr/blob/592ac781672f07eb67e935d4155570c5960d1fdb/R/get_service_info.R#L14) (consulte também a seção [docs da tag roxygen2](https://roxygen2.r-lib.org/articles/rd.html?q=examplesIf#examples)).
+- Para vinhetas, pré-compile se forem necessárias ferramentas/dados/credenciais especiais que não estão disponíveis em servidores de compilação genéricos (consulte https://ropensci.org/blog/2019/12/08/precompute-vignettes/) ou use a variável `IN_PKGDOWN` com a opção knitr eval. Exemplo: 60d1fdb/vignettes/gtexr.Rmd#L16 ou:
```

Comment:
Hi! Is `60d1fdb/vignettes/gtexr.Rmd#L16` an external URL? If it is, for me is showing 404. If it's not, I'm not sure if I understood the context.

Reply:
Thanks!